modules/cli/src/main/kotlin/datamaintain/cli/app/update/db/UpdateDb.kt

```diff
@@ -32,6 +32,12 @@ class UpdateDb(runner: (DatamaintainConfig) -> Unit = ::defaultUpdateDbRunner) :
         defaultValue = CoreConfigKey.SCAN_IDENTIFIER_REGEX.default
     )
 
+    private val filenameRegex: String? by detailedOption(
+        help = "regex to filter scripts name",
+        example = "(.*)(\\.js$)",
```

Comment:
I find the groups here confusing because there is not group extraction.
```suggestion
        example = ".*\\.js$",
```